static/templates/settings/emoji_settings_warning_modal.hbs

```diff
@@ -0,0 +1,14 @@
+<div id="emoji_warning_modal" class="modal modal-bg hide fade" tabindex="-1" role="dialog" aria-labelledby="emoji_warning_modal_label" aria-hidden="true">
+    <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="{{t 'Confirm emoji ?' }}"><span aria-hidden="true">&times;</span></button>
+        <h3 id="emoji_warning_modal_label">{{t "Confirm emoji ?" }}</h3>
+    </div>
+    <div class="modal-body">
+        <p>{{#tr this}}There already exists a default emoji with a similar name.{{/tr}}</p>
+        <p>{{t "This could lead to overriding of custom emoji over the default emoji . Do you wish to continue ?" }}</p>
```

Comment:
We should make this text more clear -- the "could" and "similar" are unnecessary -- we know it will definitely override an existing emoji, right?

Reply:
Yup , How about -- "This leads to overriding of custom emoji over the default emoji. Do you wish to continue ?". Any other suggestion for the former text ?